mitodl/mitxpro
# Added digital credentials dialog and redirection

#### Pre-Flight checklist
🔴 **NOTE: The old/initial PR was closed since the brach was force deleted, So all the future communication should be moved here. For reference of comments/changes requested on the old PR you can see [this link](https://github.com/mitodl/mitxpro/pull/2167).**


#### What are the relevant tickets?
#2152 

#### What's this PR do?
Updates the UX to support the digital credentials on the dashboard for certificates.

#### How should this be manually tested?
- Create Program and Course certificates for your user after enrollment.
- Set/Enable `FEATURE_DIGITAL_CREDENTIALS ` flag.
- You should be able to see `Digital Credentials` link on the dashboard
- Click on the link and you should be able to verify the designs and functionality from the invasion links provided in the associated.
- On clicking `Download Digital Credentials` button you should be able to observe a redirection on the `deep_link_url` that we get from Digital Credentials API.
More Details are available on the associated ticket.

#### Screenshots (if appropriate)
**Dashboard(Desktop)**
<img width="1440" alt="Screenshot 2021-03-25 at 9 28 08 PM" src="https://user-images.githubusercontent.com/34372316/112508258-3736d400-8db1-11eb-8967-ea41169fa7eb.png">

**Modal (Desktop)**
<img width="1437" alt="Screenshot 2021-03-25 at 9 28 00 PM" src="https://user-images.githubusercontent.com/34372316/112508230-31d98980-8db1-11eb-8209-17073d2778ed.png">

**Dashboard(Mobile)**
<img width="499" alt="Screenshot 2021-03-25 at 9 28 23 PM" src="https://user-images.githubusercontent.com/34372316/112508274-39009780-8db1-11eb-8c4e-34f9151eda3a.png">

**Modal (Mobile)**

<img width="501" alt="Screenshot 2021-03-26 at 4 39 07 PM" src="https://user-images.githubusercontent.com/34372316/112626305-d9a69400-8e51-11eb-9fbe-1e52edd7b1d0.png">
